RecoEcal/EgammaClusterAlgos/src/SCEnergyCorrectorSemiParm.cc

```diff
@@ -51,6 +51,7 @@ SCEnergyCorrectorSemiParm::SCEnergyCorrectorSemiParm()
       caloGeom_(nullptr),
       isHLT_(false),
       isPhaseII_(false),
+      regTrainedWithPS_(true),
```

Comment:
@ram1123 please change this to `false`, see discussion at https://github.com/cms-sw/cmssw/pull/37491#pullrequestreview-935418113